### ISSUE TITLE
When trying to move by page in a microsoft word document, nvda doesn't read page numbers. I looked in the format settings dialog and found that the report pages was checked but it stilol doesn't read page numbers when trying to move by page in microsoft word.

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#233]
**Reported by kenny11 on 9 Aug 2008 20:22 UTC**
When pressing control plus page down and control plus page up nvda will not report which page you are on.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 9 Aug 2008 23:58 UTC**
Duplicate of #120.
